dcs/src/main/java/org/trafodion/dcs/master/ServerManager.java

```diff
@@ -240,15 +240,14 @@ public ScriptContext call() throws Exception {
                         }
                     }
                 } else {
-                    if (LOG.isDebugEnabled())
-                        LOG.debug("No restart for "
-                                + znodePath
-                                + "\nbecause DcsServer start time ["
-                                + DateFormat.getDateTimeInstance().format(
-                                        new Date(serverStartTimestamp))
-                                + "] was before DcsMaster start time ["
-                                + DateFormat.getDateTimeInstance().format(
-                                        new Date(startupTimestamp)) + "]");
+                    LOG.info("No restart for "
```

Comment:
Looks like there is another reason for no restart, namely (master.isFollower() && runningServers.size() < configuredServers.size()) ?

Reply:
yes . if backup-master take over from the failed master, then the 'isFollower' is true, the result will be the code does not run the if conditions, and dcs server not restart

Reply:
RIght... My point was that the text in the logging seems to need updating, because the 'if' conditions have changed. What do you think?

Reply:
You are right, I should change the log, thanks.